src/aliases/store/postgres/alias.go

```diff
@@ -0,0 +1,58 @@
+package aliaspg
+
+import (
+	"context"
+	goerrors "errors"
+
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	models "github.com/consensys/quorum-key-manager/src/aliases/store/models"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
+)
+
+var _ aliasstore.Alias = &Alias{}
+
+// Alias stores the alias data in a postgres DB.
+type Alias struct {
+	pgClient postgres.Client
+}
+
+func NewAlias(pgClient postgres.Client) *Alias {
```

Comment:
As we design it models are never exposed outside of this layer and we are using entities as our business object representation and as DTO (data transfer object) so every layer of our project communicates using them and each layer might have its internal respresentation to communicate with the outside world (DB, KeyVault, Kafka....).

Doing that we prevent our Business component, such as our `Connectors` to be impacted by changes in our DB model

Reply:
Done, and I even added the business layer with the entities.